packages/create-invoice-form/src/lib/invoice/form.svelte

```diff
@@ -175,6 +175,16 @@
       return invoiceCurrency.hash === currency.hash;
     });
   }
+
+  $: if (!formData.issuedOn) {
+    formData.issuedOn = inputDateFormat(new Date());
+  }
+
+  $: if (!formData.dueDate) {
+    formData.dueDate = inputDateFormat(
+      new Date(new Date(formData.issuedOn).getTime() + 24 * 60 * 60 * 1000)
```

Comment:
Potential issue: mismatched variable - checks `formData.dueDate` but then calculates using `formData.issuedOn`. Please fix or explain.

Reply:
The code is actually working as intended. Let me explain the logic:
- This is a reactive statement ($:) that runs whenever its dependencies change
- It checks if formData.dueDate is empty/null/undefined
- If there's no due date set, it calculates a default due date based on formData.issuedOn by adding 24 hours (1 day)

This is a common pattern for setting a default due date that's one day after the issue date. The logic is sound because:
- formData.issuedOn is guaranteed to have a value (there's another reactive statement above it that sets a default value)
- It only sets the due date if one hasn't been specified

Using issuedOn as the base for calculation makes sense since the due date should be relative to when the invoice was issued